### PR TITLE
chore(example): Use `@nuxt/ui` for examples

### PR DESCRIPTION
**Pull request by antfu**
Currently migrated:

- [x] useCookies
- [x] locale
- [x] components 

We could either merge it first and land in multiple PRs, or wait until we have more.

###### Previews

Screen Shot 2021-12-09 at 11 09 33 PM
Screen Shot 2021-12-09 at 11 09 47 PM
Screen Shot 2021-12-09 at 11 09 01 PM
Screen Shot 2021-12-10 at 5 06 57 PM

### CODE REVIEW

File: examples/use-cookie/app.vue [hunk 1]
```diff
@@ -1,30 +1,78 @@
...
+            </NButton>
+          </div>
+        </template>
+      </NCard>
+
+      <!-- Tips -->
+      <div :class="interacted ? 'opacity-100' : 'opacity-0'" class="transition py-5 flex items-center gap-2 text-gray-400">
+        <NIcon icon="carbon-idea" class="text-xl flex-none" />
+        <div>
+          This demo showcases you how to use the
```
Comment (danielroe) [inline examples/use-cookie/app.vue:68]:
```suggestion
          This demo showcases using the
```

File: examples/use-cookie/app.vue [hunk 2]
```diff
@@ -1,30 +1,78 @@
...
+          <a href="https://v3.nuxtjs.org/docs/usage/cookies" class="bold hover:text-green5 hover:underline">useCookie</a>
+          API to persistent small amount of data that can be used by both client side and server side.
```
Comment (danielroe) [inline examples/use-cookie/app.vue:70]:
```suggestion
          API to persist small amounts of data that can be used both client-side and server-side.
```

File: examples/locale/app.vue [hunk 3]
```diff
@@ -1,17 +1,55 @@
...
+        </div>
+      </div>
+      <NCard class="mt-8 p-6 flex flex-col h-50 gap-2 text-center">
+        <h1 class="text-xl opacity-50">
+          Nuxt birthday
+        </h1>
+        <p class="text-4xl">
+          {{ date }}
+        </p>
+        <label for="locale-chooser">Locale</label>
```
Comment (danielroe) [inline examples/locale/app.vue:35]:
```suggestion
        <label for="locale-chooser">Preview a different locale</label>
```

**Review by danielroe — COMMENTED** [review]
This looks fantastic! A few minor wording changes.

**Review by pi0 — APPROVED** [review]

**Pull request merged**